tutorial/developing_packages/workspaces.rst

```diff
@@ -0,0 +1,285 @@
+   # They should be defined in the liba/CMakeLists.txt, but we can fix it here
+   add_library(liba::liba ALIAS liba)
+   add_project(libb)
+   add_library(libb::libb ALIAS libb)
+   add_project(app1)
+
+
+So basically, the super-project uses ``FetchContent`` to add the subfolders' sub-projects.
+For this to work correctly, the subprojects must be CMake based subprojects with
+``CMakeLists.txt``. Also, the subprojects must define the correct targets as would be
+defined by the ``find_package()`` scripts, like ``liba::liba``. If this is not the case,
+it is always possible to define some local ``ALIAS`` targets.
+
+The other important part is the ``conanws.py`` file:
+
+.. code-block:: python
+   :caption: conanws.py
+
+   from conan import Workspace
+   from conan import ConanFile
+   from conan.tools.cmake import CMakeDeps, CMakeToolchain, cmake_layout
+
+   class MyWs(ConanFile):
+      """ This is a special conanfile, used only for workspace definition of layout
+      and generators. It shouldn't have requirements, tool_requirements. It shouldn't have
+      build() or package() methods
+      """
+      settings = "os", "compiler", "build_type", "arch"
+
+      def generate(self):
+         deps = CMakeDeps(self)
+         deps.generate()
+         tc = CMakeToolchain(self)
+         tc.generate()
+
+      def layout(self):
+         cmake_layout(self)
+
+   class Ws(Workspace):
+      def root_conanfile(self):
+         return MyWs  # Note this is the class name
+
+
+The role of the ``class MyWs(ConanFile)`` embedded conanfile is important, it defines
+the super-project necessary generators and layout.
+
+The ``conan workspace install`` does not install the different editables separately, for
+this command, the editables do not exist, they are just treated as a single "node" in
+the dependency graph, as they will be part of the super-project build. So there is only
+a single generated ``conan_toolchain.cmake`` and a single common set of dependencies
+``xxx-config.cmake`` files for all super-project external dependencies.
+
+
+The template above worked without external dependencies, but everything would work
+the same when there are external dependencies. This can be tested with:
+
+.. code-block:: bash
+
+   $ conan new cmake_lib -d name=mymath
+   $ conan create .
+   $ conan new workspace -d requires=mymath/0.1
+   $ conan workspace install
+   $ cmake ...
+
+
+.. note::
+
+   The current ``conan new workspace`` generates a CMake based super project.
+   But it is possible to define a super-project using other build systems, like a
+   MSBuild solution file that adds the different ``.vcxproj`` subprojects. As long as
+   the super-project knows how to aggregate and manage the sub-projects, this is possible.
+
+   It might also be possible for the ``add()`` method in the ``conanws.py`` to manage the
+   addition of the subprojects to the super-project, if there is some structure.
+
+
+.. _tutorial_workspaces_orchestrated:
```

Comment:
I think this might be too much for the tutorial. Maybe it is enough to give the concept of the workspace, and the example of the monolithic build, and leave the orchestrated one out of the initial tutorial?